app/src/main/java/com/babylon/wallet/android/data/repository/cache/database/StateDao.kt

```diff
@@ -251,6 +251,14 @@ interface StateDao {
     )
     fun getAccountResourceJoin(resourceAddress: ResourceAddress, accountAddress: AccountAddress): AccountResourceJoin?
 
+    @Query(
+        """
+        SELECT * FROM AccountResourceJoin
+        WHERE resource_address = :resourceAddress
+    """
+    )
+    fun getAccountResourceJoin(resourceAddress: ResourceAddress): AccountResourceJoin?
```

Comment:
We need to display the amount of NFT items owned by the user in Hidden Assets screen where we don't have any reference to a specific account. We don't care what account the NFT collection belongs to, only what's the owned amount, which can be found in the `AccountResourceJoin` entity.